doc/nrf/protocols/bt/bt_qualification/index.rst

```diff
@@ -0,0 +1,124 @@
+
+Pre-qualified design
+********************
+
+.. note::
+   A *design* is a specific configuration of hardware and/or software implementation of adopted Bluetooth specifications, either in parts or as a whole.
+
+By using pre-qualified designs, you can omit the testing phase during your product's qualification.
+
+|NCS| contains qualified portions of various Bluetooth features (:ref:`ug_ble_controller_softdevice` versions).
+
+Any qualified design offered by Nordic Semiconductor (for example, :ref:`ug_bt_mesh` versions) can be qualified independently by the customers.
+However, this process may be complex and time-consuming, thus it is recommended to inherit Nordic Semiconductor's design qualifications in your End Product's listing.
+If you want to create a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_ yet, run the relevant tests through the `Profile Tuning Suite (PTS)`_ tool and qualify the product.
+
+A full list of Bluetooth product types including their descriptions is available on the `Bluetooth product types`_ page.
+
+
+Qualification process
+*********************
+
+The Bluetooth End Product qualification cannot be inherited.
+Hence, you must complete this process by yourself.
+You can either perform a full qualification process for a new product or edit an existing entry for a product that has already been qualified.
+
+The only way to qualify your product is to complete the `Bluetooth SIG's Qualification Process`_ under your member company's account.
+To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.
+
+Before you start the qualification process, review `PRD and DPD`_ documents, that will help you to understand the basics.
+
+Creating new entry
+==================
+
+Log into `Launch Studio`_, the official qualification tool, that will guide you through one of the following processes depending on the state of your product:
+
+* `Qualification process with no required testing`_ - If your complete product uses only pre-qualified End Products or Subsystems and implements APIs as instructed in the corresponding |NCS| documentation.
+* `Qualification process with required testing`_ - In the following cases:
+
+  * If your product uses a new Bluetooth design that is not yet qualified by Nordic Semiconductor.
+  * If your product uses a pre-qualified Component.
+  * If your product alters a previously qualified Bluetooth design by changing the code or configuring it in a way that can break the qualification requirements for the Nordic Semiconductor's qualified design.
+
+Updating existing entry
+=======================
+
+.. note::
+
+   An existing entry can be edited only by the person that completed the listing.
+   If you want to change the entry owner, contact SIG support.
+
+
+Update an existing qualification entry through `Launch Studio`_ in the following cases:
+
+* You changed an existing product's design.
```

Comment:
I don't understand this. In "Pre-qualified design" the gist is that designs must be qualified. In line 56, you repeat this information. Yet here I get an option to change the design just by editing the entry. So does this apply only to qualified designs? Or this is related to what you later explain in lines 76-82, in which case we need a mention here that some restrictions apply?

Reply:
Yeah, I had a similar thought here, but choose to leave it out since I understood the content anyway and readers might also understand it and thought this could be updated later if we get queries. Some cleanup here would help but, personally not a deal breaker for me.

Reply:
You can remove this bullet point or rephrase it a bit. If you do design change, most likely you will have to redo the testing because new features got added. If you do a design change that adds new features, you cannot edit existing entry. 

The only case when you could edit existing entry with the design change, per my understanding, is that you changed the design but overall functionality remained the same (for example: someone re-wrote a transport layer of the stack. It is big design change, but ultimately it still does the same things that it was doing before, may be now with better optimized code).

```suggestion
* You changed an existing product's design (but features used in the product are still the same) and you want to update the listing to indicate new updates or add new product marketing information.
```

Typically when one needs to add new variants of the products (like simple bulb, fancy bulb, intelligent bulb), they should add new product entry rows to the listing by doing "subsetting" of the main listing. Subsetting allows you to disable unsued features from individual product's ICS (https://support.bluetooth.com/hc/en-us/articles/360049019592).